core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -516,6 +517,28 @@ oppia.factory('explorationAutomaticTextToSpeechService', [
   }
 ]);
 
+oppia.factory('explorationCorrectnessFeedbackService', [
+  'explorationPropertyService', function(explorationPropertyService) {
+    var child = Object.create(explorationPropertyService);
+    child.propertyName = 'correctness_feedback_enabled';
+
+    child._isValid = function(value) {
+      return (typeof value === 'boolean');
+    };
+
+    child.isAutomaticTextToSpeechEnabled = function() {
```

Comment:
I don't think this method and the following method should be present here.

Reply:
Oops, my bad -- copy/paste error. Fixed. Thanks!